caikit/interfaces/runtime/server.py

```diff
@@ -0,0 +1,93 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file contains interfaces and descriptor functions for the runtime servers
+"""
+
+# Standard
+from enum import Enum
+import re
+
+# Local
+from caikit.config import get_config
+from caikit.interfaces.runtime.service import (
```

Comment:
We still haven't really settled on a hard-and-fast rule w.r.t. relative vs absolute imports. I prefer relative, especially for direct siblings (`from .service import ...` here)